build-tools/packages/build-cli/src/commands/info.ts

```diff
@@ -34,7 +86,7 @@ export default class InfoCommand extends BaseCommand<typeof InfoCommand> {
 
 	static readonly enableJsonFlag: boolean = true;
 
-	async run(): Promise<PackageVersionList> {
+	async run(): Promise<Record<string, string>[]> {
```

Comment:
Aside from the new --columns option, the only behavioral change to 'flub info' is that JSON now looks like this:
```
[
  ...
  {
    "releaseGroup": "server",
    "name": "tinylicious",
    "private": "",
    "version": "5.0.0"
  }
]

Instead of:
```
{
    ...
      "tinylicious": "5.0.0"
}
```
...which is a little less convenient if you were looking up packages by name.  LMK if you'd like something different.

Reply:
JSON changes are fine, but did you mean "private" to be a string instead of a bool?

Reply:
I was preserving what was originally displayed in the table, but now that you mention it, I agree it would be nicer to have a boolean for the JSON output.  (Fixed)

(In the table, displaying custom strings like '-private-' makes it easier to guess what's in the column without scrolling up to the header.  I think that's a nice touch.)